### DISCUSSION TITLE
Starting Community Triagers process in our repo

### DISCUSSION THREAD

**Discussion by PureWeen** [dotnet/maui#24649] [Announcements]
I want to take a moment to express our gratitude to the .NET MAUI community for their contributions. We appreciate the effort put forth by our community members in logging issues, submitting pull requests, and most importantly, helping us maintain a welcoming and safe environment for all, especially newcomers.

As a next step, we are granting a select group of community members the ability to assist with triaging issues and pull requests in our repository. This will enable them to apply / change labels on issues and PRs as well as close issues after answering them or marking them as duplicates. The following community member has agreed to be included: @kubaflo. @kubaflo has a proven track record of contributions and consistently demonstrate a helpful and welcoming attitude towards others.

We have seen great success with this "triagers" process in the dotnet/runtime repo, and we believe it will be beneficial here as well. We plan to give these permissions to more community members over time, as we continue to work towards improving our open-source project.

We welcome any feedback or suggestions on how we can better support and recognize our community here. Thank you for your continued support.

**Comment by kubaflo** [comment]
Thank you!

**Comment by AmSmart** [comment]
Amazing

**Comment by ewerspej** [comment]
Well deserved and a great step forward for the development and improvement of MAUI.
